src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+@import "./variables";
+
+.pagecontent {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 40%;
```

Comment:
You can't set width to such a value. It needs to be calculated basing on your variables.
HINT: 7 * size of cell (don't forget about gaps)